detox/test/integration/initialization-test.test.js

```diff
@@ -1,15 +1,16 @@
-const { execCommand } = require('./utils/exec');
+const execa = require('execa');
 
 describe('Initialization (context) tests', () => {
   test.each([
     `cross-env DETOX_CONFIGURATION=stub node integration/utils/simplistic-runner.js`,
 
-    `detox test -c stub --config integration/e2e/config.js --runInBand --retries 1 stub1 stub2`,
-    `cross-env DETOX_CONFIGURATION=stub jest --config integration/e2e/config.js --runInBand stub2 stub3`,
+    `detox test -c stub --config integration/e2e/config.js --runInBand --retries 1 flaky passing`,
```

Comment:
The test runner invocations with `--retries` include running a flaky tests.

The invocations without retries are limited to nominally passing tests.

This way we can be as strict as possible and demand the `exitCode` to be 0.

That improves the value of the `initialization-test.test.js` suite in CI.